pkgs/top-level/all-packages.nix

```diff
@@ -13208,6 +13208,10 @@ with pkgs;
 
   dex-oidc = callPackage ../servers/dex { };
 
+  doh-proxy = callPackage ../servers/dns/doh-proxy {
+    pythonPackages = python3Packages;
```

Comment:
drop this line

Reply:
Really? Every other time I’ve looked at this sort of thing in Nixpkgs (Go, for example), this has been encouraged, because it means the package doesn’t have to care what version the rest of nixpkgs is using.

Reply:
It would be bad if we imported `python36Packages` but that's not the case. Python 2/3 is a special case.

Reply:
I see. Fair enough.